src/model_signing/manifest.py

```diff
@@ -263,3 +404,24 @@ def resource_descriptors(self) -> Iterator[ResourceDescriptor]:
         """Yields each resource from the manifest, one by one."""
         for item, digest in sorted(self._item_to_digest.items()):
             yield ResourceDescriptor(identifier=str(item), digest=digest)
+
+    @property
+    def model_name(self) -> str:
+        """The name of the model when serialized.
+
+        This is the final component of the path and is only informative.
+        Changing the name of the model should still result in the same digests
+        after serialization, it must not invalidate signatures. As a result, two
+        manifests with different model names but with the same resource
+        descriptors will compare equal.
+        """
+        return self._name
+
+    @property
+    def serialization(self) -> dict[str, Any]:
+        """The serialization (and arguments) used to build the manifest.
```

Comment:
I have similar thoughts here regarding the `serialization` name not being informative enough of a name. But here it's a more public of a name.

Reply:
Changed to `serialization_type`